dist/realtime.js

```diff
@@ -693,31 +688,52 @@
      * Triggering
      */
 
-    on: function (name, handler) {
-      if (this.__bindings.hasOwnProperty(name))
-        this.__bindings[name].push(handler);
-      else
-        this.__bindings[name] = [handler];
+    on: function (name, handler, context, once) {
+      if (context === null)
+        context = undefined;
+      if (!this.__bindings.hasOwnProperty(name))
+        this.__bindings[name] = [];
+      this.__bindings[name] = this.__bindings[name].concat([
+        [handler, context, Boolean(once)]
+      ]);
     },
 
-    off: function (name, handler) {
+    off: function (name, handler, context) {
+      if (context === null)
+        context = undefined;
       if (!this.__bindings.hasOwnProperty(name))
         return;
-      var index = $.inArray(handler, this.__bindings[name]);
-      if (index !== -1)
-        this.__bindings[name].splice(index, 1);
+      this.__bindings[name] = this.__bindings[name].filter(function (listener) {
```

Comment:
As this conversation could last long, I join @matghaleb point of view. Since this lib is set as a 3rd party 'vendor' it should be the more agnostic possible (either if only used on Solid and Wisembly) if we want to integrate it in another piece of code. I find quite problematic too to ask users (even if it is only Wisembly front teams) to uses Core.js to support this piece of code. Especially when filter could be shimmed here in a few lines.